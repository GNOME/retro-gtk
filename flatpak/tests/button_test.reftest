[Retro Reftest]
Path=/button_test
Core=/app/lib/libretro/button_test_libretro.so
Controllers=Joypad;

[Options]

[Frame 0]
State=None
Video=button_test.png

[Frame 1]
Controller 0=Joypad UP;
State=None
Video=button_test_up.png

[Frame 2]
Controller 0=Joypad DOWN;
State=None
Video=button_test_down.png

[Frame 3]
Controller 0=Joypad LEFT;
State=None
Video=button_test_left.png

[Frame 4]
Controller 0=Joypad RIGHT;
State=None
Video=button_test_right.png

[Frame 5]
Controller 0=Joypad A;
State=None
Video=button_test_a.png

[Frame 6]
Controller 0=Joypad B;
State=None
Video=button_test_b.png

[Frame 7]
Controller 0=Joypad X;
State=None
Video=button_test_x.png

[Frame 8]
Controller 0=Joypad Y;
State=None
Video=button_test_y.png

[Frame 9]
Controller 0=Joypad START;
State=None
Video=button_test_start.png

[Frame 10]
Controller 0=Joypad SELECT;
State=None
Video=button_test_select.png

[Frame 11]
Controller 0=Joypad L;
State=None
Video=button_test_l.png

[Frame 12]
Controller 0=Joypad R;
State=None
Video=button_test_r.png

[Frame 13]
Controller 0=Joypad L2;
State=None
Video=button_test_l2.png

[Frame 14]
Controller 0=Joypad R2;
State=None
Video=button_test_r2.png

[Frame 15]
Controller 0=Joypad L3;
State=None
Video=button_test_l3.png

[Frame 16]
Controller 0=Joypad R3;
State=None
Video=button_test_r3.png
