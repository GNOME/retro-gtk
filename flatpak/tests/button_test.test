[Retro Reftest]
Path=/button_test
Core=/app/lib/libretro/button_test_libretro.so

[Frame 0]
Tests=Run;Video;NoStateAccess;
Video=button_test.png
