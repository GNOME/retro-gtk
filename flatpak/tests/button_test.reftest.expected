/button_test/Boot: OK
/button_test/Options: OK
/button_test/0/State None: OK
/button_test/0/Run: OK
/button_test/0/Video: OK
/button_test/1/State None: OK
/button_test/1/Run: OK
/button_test/1/Video: OK
/button_test/2/State None: OK
/button_test/2/Run: OK
/button_test/2/Video: OK
/button_test/3/State None: OK
/button_test/3/Run: OK
/button_test/3/Video: OK
/button_test/4/State None: OK
/button_test/4/Run: OK
/button_test/4/Video: OK
/button_test/5/State None: OK
/button_test/5/Run: OK
/button_test/5/Video: OK
/button_test/6/State None: OK
/button_test/6/Run: OK
/button_test/6/Video: OK
/button_test/7/State None: OK
/button_test/7/Run: OK
/button_test/7/Video: OK
/button_test/8/State None: OK
/button_test/8/Run: OK
/button_test/8/Video: OK
/button_test/9/State None: OK
/button_test/9/Run: OK
/button_test/9/Video: OK
/button_test/10/State None: OK
/button_test/10/Run: OK
/button_test/10/Video: OK
/button_test/11/State None: OK
/button_test/11/Run: OK
/button_test/11/Video: OK
/button_test/12/State None: OK
/button_test/12/Run: OK
/button_test/12/Video: OK
/button_test/13/State None: OK
/button_test/13/Run: OK
/button_test/13/Video: OK
/button_test/14/State None: OK
/button_test/14/Run: OK
/button_test/14/Video: OK
/button_test/15/State None: OK
/button_test/15/Run: OK
/button_test/15/Video: OK
/button_test/16/State None: OK
/button_test/16/Run: OK
/button_test/16/Video: OK
